.travis.yml

```diff
@@ -4,21 +4,14 @@ stages:
   - tests
 jobs:
   include:
-    - stage: prechecks
-      python: 3.6
-      env: TOXENV=pylint
     - python: 3.6
-      env: TOXENV=formatting
-    - python: pypy3
-      env: TOXENV=pypy
-    - python: 3.6
-      env: TOXENV=py36,py36-six
```

Comment:
Just noticed, we probably broke the deploy stage by removing `py36` here.

Reply:
It was already broken but not the same way 😄 If you want to fix it feel free.. I'm aiming to fix pylint's warning and finish #937 before releasing 2.5.4